features/quizmaster/running_quiz.feature

```diff
@@ -20,3 +20,17 @@ Scenario: I send the first question
   When I press the "Send" button for question "What is 2+2?"
   And I switch to window "1"
   Then I should see "What is 2+2?"
+
+Scenario: I send multiple questions
+  Given I have sent the first question
+  And I click the "Correct" button
+  When I click the "Back to Questions" link
+  And I wait for the page to load
+  And I should see "Correct"
+
+Scenario: Resetting the quiz
+  Given I have sent the first question
+  And "Craft Academy" has answered question "What is 2+2?" with "Six"
+  When I press the "Reset the Quiz" button
+  Then I should not see "Correct"
+  And there should be "0" answers for "What is 2+2?"
```

Comment:
I guess the `to_i` is still necessary then .. I assumed it'd get that as an integer.. my bad